src/main/java/mate/dao/CarDaoImpl.java

```diff
@@ -161,7 +161,7 @@ public List<Car> getAllByDriver(Long driverId) {
     private void insertAllDrivers(Car car) {
         Long carId = car.getId();
         List<Driver> drivers = car.getDrivers();
-        if (drivers.size() == 0) {
+        if (drivers == null || drivers.size() == 0) {
```

Comment:
I think It's better to initialize drivers by empty list in Car constructor

Reply:
Done